src/components/Layouts/MainLayout/MainLayout.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+  <div class="main-layout">
+    <button-translation />
+    <aside class="main-sidebar">
+      <h1 class="main-sidebar__title">
```

Comment:
Remember, this component is a layout component, if is a layout componet can not have functionailty or another components inside maybe you ca create another directory with navigation componentes and use this component only to the layout like a : 
```suggestion
      <template>
  <div class="main-layout">
    <button-translation />
    <aside class="main-sidebar">
      <h1 class="main-sidebar__title">
        <img
          :src="$props.img"
          width="32"
          height="30"
          :title="$t('common.components.sidebar.titleImgAlt')"
          :alt="$t('common.components.sidebar.titleImgTitle')"
        />
        <span class="main-sidebar__title-text">{{
          prop.title
        }}</span>
        <slot name="nav-title"> </slot>
      </h1>
      <nav class="main-sidebar__main-nav">
        <slot name="nav-content-main"> </slot>
      </nav>
      <nav class="main-sidebar__footer">
        <slot name="nav-content-footer"> </slot>
      </nav>
    </aside>
    <main class="main-layout__content">
      <slot></slot>
    </main>
  </div>
</template>
```

Reply:
Done!!! I have created a new component ‘layout’ to the sidebar and added the sidebar via the slot.

Thanks for the suggestion!